src/server/db/models/github-user.js

```diff
@@ -9,7 +9,7 @@ import db from '../index';
  *   updated_at: update date
  *   last_login_at: last login date
  */
-export const GitHubUser = db.Model.extend({
+db.model('GitHubUser', {
```

Comment:
I definitely don't like the second option.  The model name belongs with its definition, IMO.

I did think about approaches along the lines of the first option, but they seem gratuitously different from what the other bookshelf-using codebases I could find are doing, and it's not clear that they really buy us all that much ...